site/static/js/main.js

```diff
@@ -6,34 +6,34 @@ import './external-purchase-button.component.js';
 
 let slideObservers = [];
 
-function initializeWishlist() {
-  let wishlist = document.querySelector('s72-userwishlist');
-  if (!wishlist) return;
-  let originalFunction = wishlist.classList.remove;
-  wishlist.classList.remove = function (className) {
-    if (className == 's72-hide') {
-      // Hide this from view
-      wishlist.style.opacity = '0';
-
+function initializeCustomSliders() {
+  ['s72-userwishlist', 's72-continue-watching'].forEach(selector => {
+    let wishlist = document.querySelector(selector);
+    if (!wishlist) return;
+
+    // Gross.
+    let originalFunction = wishlist.classList.remove;
+    wishlist.classList.remove = function (className) {
       // Remove class
       originalFunction.apply(this, [className]);
+      if (className === 's72-hide') {
+        // Hide this from view
+        wishlist.style.opacity = '0';
+
+        /* Convert this to a swiper */
+        let containers = wishlist.getElementsByClassName('swiper-container');
+        if (containers.length > 0) {
+          let container = containers[0];
+          let swiper = initializeSwiper(container, true);
+          init(swiper);
+          toggleButtons(container);
+        }
 
-      /* Convert this to a swiper */
-      let containers = wishlist.getElementsByClassName('swiper-container');
-      if (containers.length > 0) {
-        let container = containers[0];
-        let swiper = initializeSwiper(container, true);
-
-        init(swiper);
-        toggleButtons(container);
+        // Now show it
+        wishlist.style.opacity = '1';
       }
-
-      // Now show it
-      wishlist.style.opacity = '1';
-    } else {
-      originalFunction.apply(this, [className]);
```

Comment:
I'm not understanding what this even does... just want to check that you're sure this `else` block should be removed?

Reply:
My intention was to refactor it so that it called the original `classList.remove` before the branch, seeing as it did this on both branches of the if/else anyway.

Looking again, perhaps it was very specifically setting opacity = 0 _before_ removing the .s72-hide (`display: none`) class so that the element transitioned to visible or something?

I dunno though, I would assume removing that class first would be pretty equivalent. It's not like the browser is going to suddenly render the element while that script is running.

The css around all this hacky swiper/slider stuff is really confusing too. There's lots of code and css that probably isn't even being used??

Reply:
Yeah I believe a lot of stuff was copypasted from an old template from before core-template... never understood why this stuff isn't all in Relish anyways.

Reply:
Seems like there is an old slider in Relish, and it was replaced with Swiper but that was done in the template, and a bunch of hacks added in Relish to support it.

Seems weird you wouldn't just upgrade the existing Relish component, or introduce a replacement? One of many weird choices that have been made around here...